astroid/brain/brain_typing.py

```diff
@@ -118,9 +119,7 @@ def looks_like_typing_typevar_or_newtype(node) -> bool:
     return False
 
 
-def infer_typing_typevar_or_newtype(
-    node: Call, context_itton: context.InferenceContext | None = None
-) -> Iterator[ClassDef]:
+def infer_typing_typevar_or_newtype(node, context_itton=None):
```

Comment:
```suggestion
def infer_typing_typevar_or_newtype(
    node: Call, context_itton: context.InferenceContext | None = None
) -> Iterator[ClassDef]:
```
We could keep the typing, right ?

Reply:
Well, yes :)